package.json

```diff
@@ -38,6 +38,7 @@
     "@material-ui/system": "5.0.0-alpha.34",
     "@types/react": "^17.0.11",
     "@types/react-dom": "^17.0.7",
+    "eslint-plugin-react": "^7.24.0",
```

Comment:
plz move to devDependencies

Reply:
fixed in https://github.com/DimensionDev/Maskbook/pull/3582/commits/d965ee339abbb3346d51f12ef80c8fc35956bb7a